query-engine/prisma/src/main.rs

```diff
@@ -104,6 +99,9 @@ pub struct PrismaOpt {
     /// Runs all queries in a transaction, including all the reads.
     #[structopt(long = "always_force_transactions")]
     always_force_transactions: bool,
+    /// Enables raw SQL queries with executeRaw mutation
+    #[structopt(long = "enable_raw_queries")]
+    enable_raw_queries: bool,
```

Comment:
I think it should probably stay that way for consistency, but flags with underscores always feel weird to me (I'd expect `enable-raw-queries`)

Reply:
I KNOW! But the other flags in the system already use underscores and I want to be consistent here, changing them to kebab-case would break the client.